### PR TITLE
Adding working nether worldborder feature

### PR DESCRIPTION
**Pull request by Keruj47**
Note: Be aware of the one issue

### CODE REVIEW

File: datapacks/Nomad Train/data/nomad_train/functions/tick.mcfunction [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
-execute as @a store result score @e[type=create:carriage_contraption,limit=1] nt.coordinates_Y run data get entity @e[type=create:carriage_contraption,limit=1] Pos[1]
-execute as @a store result score @e[type=create:carriage_contraption,limit=1] nt.coordinates_Z run data get entity @e[type=create:carriage_contraption,limit=1] Pos[2]
-scoreboard players set @e[type=create:carriage_contraption,limit=1] nt.Nether_multiplier 8
-scoreboard players operation @e[type=create:carriage_contraption,limit=1] nt.Nether_coordinates_X = @e[type=create:carriage_contraption,limit=1] nt.coordinates_X
-scoreboard players operation @e[type=create:carriage_contraption,limit=1] nt.Nether_coordinates_Z = @e[type=create:carriage_contraption,limit=1] nt.coordinates_Z
-scoreboard players operation @e[type=create:carriage_contraption,limit=1] nt.Nether_coordinates_X *= @e[type=create:carriage_contraption,limit=1] nt.Nether_multiplier
-scoreboard players operation @e[type=create:carriage_contraption,limit=1] nt.Nether_coordinates_Z *= @e[type=create:carriage_contraption,limit=1] nt.Nether_multiplier
\ No newline at end of file
+#Executing general function
```
Comment (Mkomko7) [inline datapacks/Nomad Train/data/nomad_train/functions/tick.mcfunction]: please add "applicable in all cases"

File: datapacks/Nomad Train/data/nomad_train/functions/the_nether.mcfunction [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#Removing forceload from previous chunk, setting marker's Pos value to Pos from nt:worldborder_center_marker storage and forceloading that chunk
```
Comment (Mkomko7) [inline datapacks/Nomad Train/data/nomad_train/functions/the_nether.mcfunction:1]: comment - please add info why forceload is needed

File: datapacks/Nomad Train/data/nomad_train/functions/load.mcfunction [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+
+#Creating scoreboards
 scoreboard objectives add nt.Nether_multiplier dummy
 scoreboard objectives add nt.coordinates_X dummy
 scoreboard objectives add nt.coordinates_Y dummy
 scoreboard objectives add nt.coordinates_Z dummy
 scoreboard objectives add nt.Nether_coordinates_X dummy
 scoreboard objectives add nt.Nether_coordinates_Z dummy
+
+#Running tick function
```
Comment (Mkomko7) [inline datapacks/Nomad Train/data/nomad_train/functions/load.mcfunction]: please add short explanation why it is called (tosimulate 1st tick of the world?)

**Review by Mkomko7 — CHANGES_REQUESTED** [review]
see comments inside the code

File: datapacks/Nomad Train/data/nomad_train/functions/tick.mcfunction (hunk 1 above)
Comment (Keruj47) [reply datapacks/Nomad Train/data/nomad_train/functions/tick.mcfunction]: done

File: datapacks/Nomad Train/data/nomad_train/functions/the_nether.mcfunction (hunk 2 above)
Comment (Keruj47) [reply datapacks/Nomad Train/data/nomad_train/functions/the_nether.mcfunction:1]: done

File: datapacks/Nomad Train/data/nomad_train/functions/load.mcfunction (hunk 3 above)
Comment (Keruj47) [reply datapacks/Nomad Train/data/nomad_train/functions/load.mcfunction]: done

**Comment by Keruj47** [comment]
> see comments inside the code

Requested changes are done

**Review by Keruj47 — COMMENTED** [review]
Made the requested changes @Mkomko7

**Review by Mkomko7 — APPROVED** [review]

**Pull request merged**